release.json

```diff
@@ -14,7 +14,7 @@
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "1.2.1",
         "WINDOWS_DDNPM_SHASUM": "f91a8875652941ba0f5b174ed77e434d2c7b8ba83f6c2499207cf8a05eb376f5",
-        "SECURITY_AGENT_POLICIES_VERSION": "master"
+        "SECURITY_AGENT_POLICIES_VERSION": "paulcacheux/sync-11-25-2021"
```

Comment:
Do we really want to set policies version to a personal branch?

Reply:
I'm sorry I should have re-drafted it, this was done for a test